<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverlapSetDialog</class>
 <widget class="QDialog" name="OverlapSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overlap 批量设定</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>指定oto.ini的位置</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="loadOtoWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="fileNameEdit_open"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="browseButton_open">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>浏览</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>加载</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="countLabel">
          <property name="text">
           <string>还未加载原音设定</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="showOtoListButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>查看列表</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="setStartWithCheckBox">
          <property name="text">
           <string>开头为以下内容之一时，将Ovl设置为</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="setStartWithSpinBox">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-99999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>11</number>
            </property>
            <property name="rightMargin">
             <number>11</number>
            </property>
            <item>
             <widget class="QRadioButton" name="setStartWithPresetRadioButton">
              <property name="text">
               <string>加载预设</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">startListButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="setStartWithPresetComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="setStartWithLoadFromFileRadioButton">
              <property name="text">
               <string>从文件加载</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">startListButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="setStartWithBrowseButton">
              <property name="text">
               <string>浏览</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="setStartWithInputRadioButton">
              <property name="text">
               <string>自定义</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">startListButtonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="setStartWithInputButton">
              <property name="text">
               <string>输入</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="setStartWithListView"/>
         </item>
         <item>
          <widget class="QCheckBox" name="matchStartOtoCheckBox">
           <property name="text">
            <string>同时也匹配开头是“- ”+上述内容的别名</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="OneThirdCheckBox">
        <property name="text">
         <string>将上述操作影响范围之外的Ovl三分之一化</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OverlapSetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OverlapSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>904</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setStartWithCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>setStartWithSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>856</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setStartWithCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="startListButtonGroup"/>
 </buttongroups>
</ui>
